src/index.d.ts

```diff
@@ -1,162 +1,192 @@
-import type { Observable, OperatorFunction, UnaryFunction } from "rxjs"
+import type { Observable, UnaryFunction } from "rxjs"
 
-interface PipeState<T> {
-  <A>(defaultOp: WithDefaultOperator<T, A>): DefaultedStateObservable<T | A>
-  <A, B>(
-    op1: OperatorFunction<T, A>,
+interface EffectOperatorFunction<T, ET, R, ER>
+  extends UnaryFunction<EffectObservable<T, ET>, EffectObservable<R, ER>> {}
+
+// prettier-ignore
+export interface PipeState<T, ET> {
```

Comment:
Script used to generate these: https://gist.github.com/voliva/fe7b6be0fd761a0b23f1194f3a928d3f